pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -53,11 +53,13 @@ type EventListener struct {
 // EventListenerSpec defines the desired state of the EventListener, represented
 // by a list of Triggers.
 type EventListenerSpec struct {
-	ServiceAccountName string                 `json:"serviceAccountName"`
-	Triggers           []EventListenerTrigger `json:"triggers"`
-	Replicas           *int32                 `json:"replicas,omitempty"`
-	NamespaceSelector  NamespaceSelector      `json:"namespaceSelector,omitempty"`
-	Resources          Resources              `json:"resources,omitempty"`
+	ServiceAccountName    string                 `json:"serviceAccountName"`
+	Triggers              []EventListenerTrigger `json:"triggers"`
+	DeprecatedServiceType corev1.ServiceType     `json:"serviceType,omitempty"`
```

Comment:
as mentioned here https://github.com/tektoncd/triggers/pull/897/files?file-filters%5B%5D=.go#r556806397 can we add comment above `DeprecatedServiceType` and mention the created issue